### DISCUSSION TITLE
Broken link in README.md

### DISCUSSION THREAD

**Discussion by dukeEllington32** [archlinux/archinstall#142] [General]
In the section of the README.md titles "This installer will perform the following," the last bullet point contains a link that 404s.

> "Installs a network-profile called **_workstation_** (more on network profiles in the docs)"

**Comment by Torxed** [comment]
Thanks for pointing it out, the `awesome` profile was called `workstation` back in the stone age, when this project was just a bash script to install my workstation at home hehe.
